src/garage/envs/__init__.py

```diff
@@ -13,6 +14,7 @@
     'GarageEnv',
     'Step',
     'EnvSpec',
+    'Environment',
```

Comment:
i think this should actually be defined in `src/garage/_environment.py` and exported as `garage.Environment`